solr/core/src/java/org/apache/solr/servlet/SolrDispatchFilter.java

```diff
@@ -533,10 +533,23 @@ public void doFilter(ServletRequest _request, ServletResponse _response, FilterC
       if (accepted) {
         rateLimitManager.decrementActiveRequests(request);
       }
+      span.setTag(Tags.HTTP_STATUS, response.getStatus());
       span.finish();
     }
   }
-  
+
+  protected Span buildSpan(HttpServletRequest request, Tracer tracer) {
+    return tracer
+        .buildSpan(
+            "http.request") // will be changed later
+        .asChildOf(tracer.extract(Format.Builtin.HTTP_HEADERS, new HttpServletCarrier(request)))
+        .withTag(Tags.SPAN_KIND, Tags.SPAN_KIND_SERVER)
+        .withTag(Tags.HTTP_METHOD, request.getMethod())
```

Comment:
do we need to add port and host as tags to keep alignment with the deleted code? or is it not necessary as it is included in the url? or were they deleted for some reason?

Reply:
I think you are observing from the diff that the span operation name used to include the host & port.  Those are poor choices for a span operation name, and this PR expressly sets out to improve this.  They weren't in dedicated tags, although HTTP_URL includes them if we're curious.